### DISCUSSION TITLE
Name and Logo

### DISCUSSION THREAD

**Discussion by Jay-Karia** [Jay-Karia/PassVault#1] [Ideas]
Here we will discuss about the name of the app and logo

**Comment by Jay-Karia** [comment]
![GettyImages-543976778](https://github.com/Jay-Karia/Password-Manager/assets/86977383/1d402cb0-67ed-465f-ae53-c188b4458291)

**Reply by Aryan9788** [reply]
Nice Logo

**Comment by Aryan9788** [comment]
PassGuardian
SecureVault
KeyLockr
PassFortify
LockBox Pro
PassSentry
VaultWarden
CipherSafe
KeyKeepHub
PassArmor
ShieldPass
CodeGuard
SafeCipher
LockVault Connect
PassLockr Plus
SecureKeyHub
PassMinder
VaultLinker
KeyCrypta
PassVault Express

**Reply by Jay-Karia** [reply]
PassVault sounds nice

**Comment by Jay-Karia** [comment]
suggest some logos

**Reply by Aryan9788** [reply]
![image](https://github.com/Jay-Karia/Password-Manager/assets/74713058/dd7d87dc-d7b4-44a0-b7fc-596fe74d864c)

**Reply by Aryan9788** [reply]
![image](https://github.com/Jay-Karia/Password-Manager/assets/74713058/bc4b48cb-1ea6-4cec-ac90-0238ba412120)

**Reply by Aryan9788** [reply]
![image](https://github.com/Jay-Karia/Password-Manager/assets/74713058/4071111e-0a82-4363-b908-a9f968474ddd)

**Reply by Jay-Karia** [reply]
2nd looks good

**Comment by Jay-Karia** [comment]
PassVault 
Final ??

**Comment by Jay-Karia** [comment]
![logo1](https://github.com/Jay-Karia/Password-Manager/assets/86977383/e3bb5be6-60a8-4c6e-ba31-87a98a076f4b)
![logo2](https://github.com/Jay-Karia/Password-Manager/assets/86977383/6c31c5d4-c326-4ad3-87c7-c603f55048dc)

**Reply by Aryan9788** [reply]
Both are good

**Comment by Jay-Karia** [comment]
![PassVault](https://github.com/Jay-Karia/Password-Manager/assets/86977383/157022e8-11f5-48d9-afe6-be5df331865a)

**Reply by Jay-Karia** [reply]
Final?

**Comment by Jay-Karia** [comment]
This is the final Logo
![PassVault_2](https://github.com/Jay-Karia/PassVault/assets/86977383/a8a779a3-7351-48a4-b854-f9b6867b9184)
